<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PairingTrustWidget</class>
 <widget class="QWidget" name="PairingTrustWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Trust</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="PortableIDWidget" name="widgetParticipantCertificate" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelStats">
     <property name="text">
      <string>Stats: N/A</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TrustSymbolWidget" name="widgetTrustLevel" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="svgURL" stdset="0">
      <string>:/icons/block.svg</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select trust level:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonTrustUnset">
       <property name="text">
        <string>Unset</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupTrust</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonTrustSet">
       <property name="text">
        <string>Trust</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupTrust</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonTrustBlock">
       <property name="text">
        <string>Block</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroupTrust</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonSaveEdits">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRemove">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../libres/resources/icons.qrc">
         <normaloff>:/icons/trash.svg</normaloff>:/icons/trash.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PortableIDWidget</class>
   <extends>QWidget</extends>
   <header location="global">node/PortableIDWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TrustSymbolWidget</class>
   <extends>QWidget</extends>
   <header location="global">pairing/TrustSymbolWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../libres/resources/icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupTrust"/>
 </buttongroups>
</ui>
